src/tokens.ts

```diff
@@ -158,6 +169,9 @@ export function tokensFor(d: Dialect) {
     if (inString(next, Space)) {
       while (inString(input.next, Space)) input.advance()
       input.acceptToken(whitespace)
+    } else if (next == Ch.Dollar && input.next == Ch.Dollar) {
```

Comment:
Shouldn't this check for the option to be enabled?

Reply:
ah, yes, you're right. Thanks